### ISSUE TITLE
Installation error in cluster

### ISSUE THREAD

**Issue by alberthxf** [leetmaa/KMCLib#1]
Hi, I succeeded in installing it on my own redhat. But when installing on cluster which have a lot c++, mpi modules, it failed. I doubt the reason comes from cmake. The make file used the /user/bin/c++ instead of provided gcc path. Could you fix this error or tell me how to fix it? 

Best,
Albert

**Comment by leetmaa** [comment]
Did you follow the installation manual? http://leetmaa.github.io/KMCLib/manual-v1.1/install.html If you try to be a bit more specific in your question I might be able to help you. But if you have MPI related installation problems on a cluster you are probably better off asking your cluster admin for help.

**Comment by alberthxf** [comment]
Yes. I followed the manual. But I also failed in building serial one. I am wondering how CMKAE know what g++ to use. Seems I can’t assign g++ library to CMAKE.


On Nov 30, 2015, at 11:07 AM, Mikael Leetmaa <notifications@github.com<mailto:notifications@github.com>> wrote:


Did you follow the installation manual? http://leetmaa.github.io/KMCLib/manual-v1.1/install.html If you try to be a bit more specific in your question I might be able to help you. But if you have MPI related installation problems on a cluster you are probably better off asking your cluster admin for help.
